metals/src/main/scala/scala/meta/metals/Configuration.scala

```diff
@@ -16,44 +16,41 @@ import Configuration._
     scalafmt: Scalafmt = Scalafmt(),
     scalafix: Scalafix = Scalafix(),
     search: Search = Search(),
-    hover: Hover = Hover(),
-    highlight: Highlight = Highlight(),
-    rename: Rename = Rename(),
+    hover: Enabled = Enabled(true),
+    highlight: Enabled = Enabled(false),
+    rename: Enabled = Enabled(false),
 )
 
 object Configuration {
   implicit val circeConfiguration: CirceConfiguration =
     CirceConfiguration.default.withDefaults
 
+  @JsonCodec case class Enabled(enabled: Boolean)
+
   @JsonCodec case class Sbt(
       enabled: Boolean = false,
-      command: String = "test:compile"
+      diagnostics: Enabled = Enabled(true),
+      command: String = "test:compile",
```

Comment:
Should this be empty?

Reply:
Yes, I didn't change it yet. Did you see https://github.com/scalameta/metals/pull/263#discussion_r179964441?

Reply:
I had seen that comment in my email but failed to find it in the github ui, thanks for the link!